components/01-atoms/forms/checkbox/checkbox.twig

```diff
@@ -1,15 +1,15 @@
 {% if not attributes %}
-<div class="form-item form-item--checkbox">
+<fieldset class="form-item form-item--checkbox">
   <legend class="h3">Options as Checkboxes</legend>
   <ul class="form-item--checkboxes">
 {% endif %}
-      {% for checkbox in checkboxes %}
+      {% for key, checkbox in checkboxes %}
```

Comment:
Unrelated, but indentation is way off here

Reply:
Fixed